Source/CheatSheet/cheatsheet.md

```diff
@@ -1,32 +1,32 @@
-# Foundation of CQL
+# CQL Cheat Sheet
 
-CQL is a Health Level 7 Standard for the expression of clinical knowledge.  
+Clinical Quality Language [(CQL)](http://cql.hl7.org) is a Health Level 7 Standard for representing clinical logic. This cheat sheet provides a quick reference for the most commonly used features of the language. For more resources, see the [Getting Started](https://github.com/cqframework/CQL-Formatting-and-Usage-Wiki/wiki/Getting-Started) page of the CQL Formatting and Usage Wiki.
 
 ## Syntax
 
 ### [**Declarations**](https://cql.hl7.org/02-authorsguide.html#declarations)
 
-Constructs expressed within CQL are packaged in containers called libraries. Each library allows a set of declarations to provide information about the library as well as to define constructs that will be available within the library.
+Constructs expressed within CQL are packaged in libraries. Each library consists of a set of declarations such as terminology, expressions, and functions that are used to define and share clinical logic.
```

Comment:
Each library 'consists of' a set of declarations. I like 'allows' better because it helps me understand what the declarations are doing. I would also like to include the bit about the declarations helping one know key information about the library in use. This is my suggested phrase:

Constructs expressed within CQL are packaged in libraries. Each library allows a set of declarations such as terminology, expressions, and functions that are used to provide information about the library and define constructs that will be available to define and share clinical logic.